### ISSUE TITLE
Constants in a switch/case statement not marked as constant

### ISSUE THREAD

**Issue by adambuczynski** [atom/language-javascript#422]
#E.g.:

![image](https://cloud.githubusercontent.com/assets/490562/18069626/80fd0818-6e9c-11e6-9532-228f86d1e325.png)

The constant `FOOD` should be marked as `constant.other.js`.

Seems to be working correctly in Github's markdown code highlighter:

```js
const SOME_CONSTANT = 1;
switch (type) {
  case SOME_CONSTANT:
    doStuff();
    break;
}
```

**Comment by 50Wliu** [comment]
Working for me using language-javascript@0.120.0.

**Comment by adambuczynski** [comment]
I'm on 119, but rebuilding Atom is having issues and throwing a `Error: Cannot find module 'fs-extra'` from the build script. I'll try again later when I can get a new build going.

**Comment by Alhadis** [comment]
Dunno what you're doing, but it sounds like you forgot to `apm install` somewhere.

**Comment by adambuczynski** [comment]
I do what I always do. Pull the latest changes from the Atom master branch, and run `script/build`. Has always worked, I've built Atom like that for 12-15 times now, but it doesn't work anymore. Where do I need to run `apm install` and why? I am just following the build instructions for OS X.

**Comment by adambuczynski** [comment]
I've raised an issue for it on the Atom repo: https://github.com/atom/atom/issues/12552 
It seems the windows/linux builds are also having some issues.

**Comment by 50Wliu** [comment]
0.120.0 isn't on Atom master yet.

**Comment by adambuczynski** [comment]
When is language-javascript 0.120.0 expected to land in Atom master branch?

**Comment by 50Wliu** [comment]
I was hoping to get it into the 1.11.0 beta, but mysteriously failing archive-view specs prevented that from happening.  There's no ETA - it depends on when those specs are fixed.

**Comment by MaximSokolov** [comment]
You can install v0.120.0 via:
```
apm install language-javascript@0.120.0
```

**Comment by adambuczynski** [comment]
@50Wliu yep, it seems fixed in 0.120.0. Thank you.
